src/makeInfinityAdder.js

```diff
@@ -1,10 +1,27 @@
 'use strict';
 
 /***
+ *
  * @return {function}
  */
+
 function makeInfinityAdder() {
-  // write code here
+  let result = 0;
+
+  const adder = (arg = 0) => {
```

Comment:
The default parameter `arg = 0` is not necessary and can be removed. It will always override falsy values, including `0`, which should be a valid input to add to the sum.